spec/ledger_spec.cr

```diff
@@ -1,6 +1,33 @@
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at # http://mozilla.org/MPL/2.0/
+
 require "./spec_helper"
 
 describe Ledger::Pow do
+  let(:txn) do
+    Ledger::Action::Transaction.new(
+      from: "me",
+      to: "you",
+      amount: 1_u64
+    )
+  end
+  let(:transactions) { [txn] of Ledger::Action::Transaction }
+  let(:block) { Ledger::Pow.mine(transactions) }
```

Comment:
what does the colon `:` do here? curious about the syntax.

Reply:
it makes the bareword `block` a symbol.
The let function turns it into a method with the name block.

Reply:
https://github.com/ysbaddaden/minitest.cr

Also it's lazy: http://ruby-journal.com/minitest-let-is-lazy/